front/src/applications/editor/Editor.scss

```diff
@@ -142,6 +142,10 @@
     & > .map {
       flex-grow: 1;
 
+      .map-modal {
+        max-width: 100%;
```

Comment:
![image](https://github.com/osrd-project/osrd/assets/6003856/015015f4-d3ee-49e7-a7e8-ad556490ea75)

It's a bit too much no ? :laughing: 
Try to define some dynamic value, you could fine inspiration on https://blog.logrocket.com/flexible-layouts-without-media-queries/ by example

Reply:
Fixed !